src/acuvity/guard/constants.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+from typing import Optional
+
+
+class ComparisonOperator(Enum):
+    """Valid comparison operators for thresholds"""
+    GREATER_THAN = '>'
+    GREATER_EQUAL = '>='
+    EQUAL = '=='
+    LESS_EQUAL = '<='
+    LESS_THAN = '<'
+
+class GuardName(Enum):
+    PROMPT_INJECTION = "prompt_injection"
+    JAIL_BREAK = "jail_break"
+    MALICIOUS_URL = "malicious_url"
+    TOXICITY = "toxicity"
+    BIAS = "bias"
+    HARMFUL_CONTENT = "harmful"
+    GIBBERISH = "gibberish"
+    LANGUAGE = "language"
+    MODALITY = "modality"
+    PII_DETECTOR = "pii_detector"
+    SECRETS_DETECTOR = "secrets_detector"
+    KEYWORD_DETECTOR = "keyword_detector"
```

Comment:
not entirely sure why my previous reviews aren't showing anymore in GitHub, but we should also have a way to list all guards / guard names.